web/src/lib/utils/helpers.ts

```diff
@@ -112,6 +112,18 @@ export const scrollToTopOfElement = (
   }, 100);
 };
 
+export const scrollAccordionToTop = (
```

Comment:
I moved `scrollAccordionToTop` to `helpers.ts` to live with the other utility functions (to be more in line with the function's function (see what I did there...lol)).

Reply:
In my personal opinion, I don't think this belongs in `helpers`. This function is called `scrollAccordionToTop` and includes a variable called `timeForAccordionToOpen`. Basically, everything in here is specific to the accordion. So I don't foresee any other components in the code calling the "scroll accordion" function. And since it's not actually shared code, I think it should stay in the same place with the accordion itself